### PR TITLE
Meetings

### PR DESCRIPTION
**Pull request by katcipis**
Some notes I made about meetings because lately I have been exposed to them after a long time of peace, made me think a little about it.

### CODE REVIEW

File: notes/meetings.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+time and you will end up with a suboptimal solution (specially
+if you need to deliver something soon, you can't wait for another
+inspired moment).
+
+The only way to avoid this is to have an average performance
+all the time, like a robot. Since humans are not robots I don't
+think anyone is capable of this regarding creative work. If your
+work is repetitive and does not require imagination/creative effort,
+then having control over your schedule to capture inspiration
+is less important, it is more easy to be interrupted and then
```
Comment (vitorarins) [inline notes/meetings.md]: *easier

File: notes/meetings.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+go back to work.
+
+In the end "creative work" ends up being tasks
```
Comment (vitorarins) [inline notes/meetings.md]: Just a suggestion, rephrase it as 'So "creative work" ends up being tasks...'

File: notes/meetings.md (hunk 1 above)
Comment (katcipis) [reply notes/meetings.md]:
```suggestion
is less important, it is easier to be interrupted and then
```

File: notes/meetings.md (hunk 2 above)
Comment (katcipis) [reply notes/meetings.md]:
```suggestion
So "creative work" ends up being tasks
```

**Comment by katcipis** [comment]
@vitorarins actually this started more like a personal note/dump of some ideas, do you think it would be useful as a published post ? (the order of the ideas are useful enough ?)

**Comment by cadicallegari** [comment]
Excellent content and great references. It gives a lot of food for thoughts. 

Thanks for sharing =)

**Comment by hlmerscher** [comment]
> @vitorarins actually this started more like a personal note/dump of some ideas, do you think it would be useful as a published post ? (the order of the ideas are useful enough ?)

@katcipis for me it is totally worth to publish somewhere! There are many relevant aspects.

**Comment by vitorarins** [comment]
> do you think it would be useful as a published post ?

I would publish as a short post (no need to get formal and fancy), it's always nice to find people going through the same pain as you :smile: 

> (the order of the ideas are useful enough ?)

But since you are asking, only thing I would add is a proper conclusion.
What do you think?

**Comment by katcipis** [comment]
> > do you think it would be useful as a published post ?
> 
> I would publish as a short post (no need to get formal and fancy), it's always nice to find people going through the same pain as you 
> 
> > (the order of the ideas are useful enough ?)
> 
> But since you are asking, only thing I would add is a proper conclusion.
> What do you think?

Makes sense, it lacks a conclusion indeed. When I get to it I will make sure to add you guys to the PR =).

Thanks @hlmerscher @vitorarins @cadicallegari

**Pull request merged**